lib/susedistribution.pm

```diff
@@ -149,13 +156,9 @@ sub x11_start_program {
     wait_still_screen(3) unless ($args{no_wait} || ($args{valid} && $args{target_match}));
```

Comment:
What do you mean with "same behavior"? The wait should *not* be triggered when we have a target match because then we have something explicit to check for and don't need to wait for a still screen. the `wait_still_screen` is only triggered "unless no_wait" == "if wait" explicitly requested or if valid or target_match have been overwritten, e.g. as used in some occassions by me with `x11_start_program('foo', valid => 0);`
*But* it is true that there is currently no call to `x11_start_program(…, no_wait => 1)` anymore but I saw no reason to delete that argument again. Should I do that?